misc/extension_api_validation/4.2-stable.expected

```diff
@@ -8,6 +8,13 @@ Add new entries at the end of the file.
 
 ## Changes between 4.2-stable and 4.3-stable
 
+GH-84472
```

Comment:
This should be at the end of the file, see above

Reply:
fixed

Reply:
You need to rebase and fix the conflict as the file has been modified by others